src/common/user-store.ts

```diff
@@ -59,6 +61,8 @@ export class UserStore extends BaseStore<UserStoreModel> {
     colorTheme: UserStore.defaultTheme,
     downloadMirror: "default",
     downloadKubectlBinaries: true,  // Download kubectl binaries matching cluster version
+    allowAutoUpdates: false,
```

Comment:
Default should be `true`.

Reply:
Doesn't that defeat the purpose of this PR? Especially since the logic of this PR to auto update and quit immediately upon finding an update.

Reply:
Wait, I thought we should just show a notification and allow user to quit+update from there? Why would we want to quit immediately if update is found?

Reply:
So if you look at the code, when this option is false then a notification with "Yes install now", "Yes install later", and "No" is displayed to the user. If a user wants to always install immediately then they would toggle this.

Reply:
Ok, got it... name confused me. Could we then remove this option completely (I think notification is enough)?

Reply:
Yeah sure, I thought that we wanted auto updates (meaning installing too) which is why I added this.